src/Azure/AzureAppServices.HostingStartup/src/Microsoft.AspNetCore.AzureAppServices.HostingStartup.csproj

```diff
@@ -7,6 +7,7 @@
     <TargetFrameworks>netcoreapp2.1;netcoreapp2.0;net461</TargetFrameworks>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <PackageTags>aspnetcore;azure;appservices</PackageTags>
+    <IsPackageInThisPatch>true</IsPackageInThisPatch>
```

Comment:
This is to make sure everything passes in CI build.

Reply:
Why was this necessary? Are we really releasing a patch of this package? If so, can you  list this package in eng/PatchConfig.props instead?

Reply:
Can you remove this before we merge?